releasenotes/notes/deferred_cluster_creation.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+releaseNotes:
+- |
+  **Added** an experimental feature to enable cluster creation on worker threads inline during requests.
+  This will save memory and CPU cycles in cases where there are lots of inactive clusters and > 1 worker thread.
+  This can be enabled by setting ENABLE_DEFERRED_CLUSTER_CREATION to true in Istiod Deployment.
```

Comment:
This is actually in the agent not istiod?

Reply:
Then the feature flag should have its own home not under pilot

Reply:
Oh yes. I forgot just copy pasted some other flag :-)

> Then the feature flag should have its own home not under pilot

We will have to do that refactor